cmd/rhoas/main.go

```diff
@@ -94,12 +95,33 @@ func generateDocumentation(rootCommand *cobra.Command) {
 	}
 }
 
-func initConfig(f *factory.Factory) {
+func initConfig(f *factory.Factory, logger logging.Logger) {
+	rhoasCfgDir, err := config.DefaultDir()
+	if err != nil {
+		logger.Errorf("Error fetching config dir.", err)
+	}
+
+	customFolder, isCustomFolderSet := os.LookupEnv("RHOASCONFIG")
+	if isCustomFolderSet {
+		rhoasCfgDir = customFolder
```

Comment:
`RHOASCONFIG` actually points to the file itself, not the directory. So it could be `$HOME/rhoascustomfile.json`. Here you are assigning it to be the holding folder, and so later it is creating the folder called `$HOME/rhoascustomfile.json`

Reply:
I had that and changed it as I thought it will be folder.